packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -680,7 +714,9 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     }
 
     if (this.state.open && _.isNumber(changes.highlightedIndex)) {
+      const isFromKeyboardItem = changes.type !== Downshift.stateChangeTypes.itemMouseEnter
```

Comment:
could you, please, just clarify our assumptions here? Am I right that

- if we have state change for `Dropdown`, which is change for its `open` value
- and this `open` value is `true`
- and, if also we have `highlightedIndex` being number in changes (does it imply that this has been changed as well - I mean, otherwise it would be non-defined, right?)

then we providing `itemIsFromKeyboard` value to item, which is an indicator of item being selected from keyboard?

Reply:
It goes like this:
- if we have state change for Dropdown and the dropdown is open
- if the highlighted index was changed
then we are providing the itemIsFromKeyboard to the DropdownItem (if it is from mouse, this will be false, otherwise it will be true)